wordpress-mobile/WordPress-Android
# Add the remaining view all screens

This PR adds the view-all screen support for the following stats types:
- Publicize
- Authors
- Countries
- Search terms
- Videos

To test:
1. Open a site that contains enough stats data for all of the stats types above
2. Go to Stats
3. Notice all the stats types contain the View more button
4. Tap on the View more button
5. Verify that additional data of that type is displayed
